docs/Documentation/Configuration/Version-Changes/Migration.md

```diff
@@ -569,3 +569,22 @@ events:
   giveSkillXP: auraskillsxp farming 5
 ```
 </div>
+
+
+### 2.2.0-DEV-89 - Rename Fabled :gear:
```

Comment:
That integration is automated, so it should get the message for it and be in a spoiler (that is also missing for the section above).